build.rs

```diff
@@ -91,6 +91,11 @@ fn produce_version_db() -> Result<JuliaupVersionDB> {
                 format!("{}+0~x86", v),
                 JuliaupVersionDBVersion {url: format!("https://julialang-s3.julialang.org/bin/linux/x86/{}.{}/julia-{}-linux-i686.tar.gz", v.major, v.minor, v)}
             );
+        } else if target_os == "linux" && target_arch == "aarch64" {
+            db.available_versions.insert(
+                format!("{}+0~x64", v),
+                JuliaupVersionDBVersion {url: format!("https://julialang-s3.julialang.org/bin/linux/aarch64/{}.{}/julia-{}-linux-aarch64.tar.gz", v.major, v.minor, v)}
```

Comment:
So the idea here is that the platform in the `format!` call and the one in the URL always match here, so in this case it should be `format!("{}+0~aarch64", v)`, I think.

For Mac and Windows if the `target_arch` is `aarch64`, I currently just ship the x64 (or was it x86?) builds and then let emulation on those platforms handle things. But that is only the case because we don't have native arm builds for Windows and Mac yet (or at least not for a released version).